src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariants.java

```diff
@@ -84,6 +84,16 @@
  * A new VCF file containing the selected subset of variants.
  * </p>
  *
+ * * <h3>Usage example</h3>
+ * <pre>
+ *     ./gatk-launch \
+ *     SelectVariants \
+ *     -R reference.fasta \
```

Comment:
GATK4 doesn't generally require that a reference be provided except in the case where the tool specifically requires it, or the input or output is a CRAM), so I'm wondering if we should remove these from the examples.